debian/homeserver.yaml

```diff
@@ -619,3 +640,60 @@ enable_group_creation: false
 #
 #user_directory:
 #   search_all_users: false
+
+
+# User Consent configuration
+#
+# Parts of this section are required if enabling the 'consent' resource under
+# 'listeners', in particular 'template_dir' and 'version'.
+#
+# 'template_dir' gives the location of the templates for the HTML forms.
+# This directory should contain one subdirectory per language (eg, 'en', 'fr'),
+# and each language directory should contain the policy document (named as
+# '<version>.html') and a success page (success.html).
+#
+# 'version' specifies the 'current' version of the policy document. It defines
+# the version to be served by the consent resource if there is no 'v'
+# parameter.
+#
+# 'server_notice_content', if enabled, will send a user a "Server Notice"
+# asking them to consent to the privacy policy. The 'server_notices' section
+# must also be configured for this to work.
+#
+# 'block_events_error', if set, will block any attempts to send events
+# until the user consents to the privacy policy. The value of the setting is
+# used as the text of the error.
+#
+# user_consent:
+#   template_dir: res/templates/privacy
```

Comment:
What is the base path inside your server for this template_dir?

Where do I have to put the en/1.0.html on my server, so it will be served correctly?

Also here is missing, how the consent url will be  served?

Reply:
1. I found out, that the base for the path must be your .synapse folder on your server.

2. the `consent` page is activated in your homeserver.yaml in the section

      listeners:
        # find `names - client` and add it accordingly